2022/spring/Sterne-MW@5pm/thiago-silva.md

```diff
@@ -0,0 +1,30 @@
+# Basics
+
+* Thiago
+* Minerva Student
+* CS Major, B Minor
+
+# History
+
+Once upon a time, in a far away city, across the river from his future home, a child was born. Blessed with unimaginable abilities, unrelenting **drive** towards competition and some anxiety to prevent him from being OP.
+
+## The competitions
+
+Wanted to go to public schools to compete, and went on to become the most awarded competitor in math olympiads fromm the entire region [national second at 11yo](http://premiacao.obmep.org.br/2011/verRelatorioPremiadosOuro.do.htm). As well as the best ranked in areas such as astronomy and linguistics.
+
+## The fall
+
+Unfortunately, he got addicted and wasted years.
+
+1. Playing league of legends
+2. As a mono champion to make it even worse
+
+## Phoenix Rising
+
+After drifting through life, he decided to be the best human who ever lived. The strongest, smartest, and most humble person to walk this earth.
+
+Beat saber player, Blockchain enjoyer 
+
+# Reviews
+
+>"Such a handsome boy" - Grandma
```

Comment:
MARKDOWN BEST PRACTICE IS TO ADD A NEW LINE AT THE END!!!!!!!!! REDOOO THISSSS!!!!!